background/session.ts

```diff
@@ -65,15 +65,15 @@ constructor (onOpen, onMessage)
 }
 
 startup () {
-
+        const defaultWebSocketPort = 12546;
```

Comment:
Good idea. I've now made this change in a separate commit rather than ask you to isolate this part of the PR.